plugin/src/main/java/com/h3xstream/findsecbugs/taintanalysis/TaintFrameModelingVisitor.java

```diff
@@ -190,6 +190,8 @@ public void visitSIPUSH(SIPUSH obj) {
 
     @Override
     public void visitGETSTATIC(GETSTATIC obj) {
+        // Scala uses some classes to represent null instances of objects
+        // If we find one of them, we will handle it as a Java Null
         if (obj.getLoadClassType(getCPG()).getSignature().equals("Lscala/collection/immutable/Nil$;")) {
```

Comment:
As of right now, the Scala Anorm injection-sink for SimpleSql.as would be using this.

When you use a StringContext without any parameters ( `s"test string"` ), the generated Byte code will call StringContext.s with a Nil object. 
`INVOKEVIRTUAL scala/Predef$.wrapRefArray ([Ljava/lang/Object;)Lscala/collection/mutable/WrappedArray;
    INVOKESPECIAL scala/StringContext.<init> (Lscala/collection/Seq;)V
    GETSTATIC scala/collection/immutable/Nil$.MODULE$ : Lscala/collection/immutable/Nil$;
    INVOKEVIRTUAL scala/StringContext.s (Lscala/collection/Seq;)Ljava/lang/String;`

I tried adding the Nil object to the IMMUTABLE types, but it doesn't seem to work. I think that since the Nil object was created using a GETSTATIC call, the taint-config is not used (I might be wrong there).